docs/getting-started.md

```diff
@@ -0,0 +1,140 @@
+﻿
+# Kubevirtci
+
+Kubevirtci rule is to create a pre provisioned clusters, publish them as a container image,\
+and allow people to run a containerized version of a k8s / open-shift cluster.
+
+Target audience of this project is Developers of kubevirt.
+
+## Getting started
+Download this repo
+```
+git clone https://github.com/kubevirt/kubevirtci.git
+cd kubevirtci
+```
+
+Make sure you have jq installed (can be installed by yum/dnf with the basic repositories).
+
+#### gocli developer guide:
+gocli is a binary that assist in provisioning and spinning (cluster-up) a cluster.
+Uses bazel as build tool.
+
+Bazel is good mostly for bigger scale projects, where it has declarative dependency tree,\
+which allows to do caching and save lot of build time,\
+also, thanks to gazelle addon, there is a rule for building each target on its own.
+
+#### Install 
+Prerequisites:
```

Comment:
Isn't there a developer doc in the project already teaching how to prepare the dev environment? This doc should reference existing docs while additionally explaining how to create a new provider.